packages/dagre-reactjs/src/types.ts

```diff
@@ -114,6 +114,7 @@ export interface DAGReactGeneralProps {
     reportSize: ReportSize
   ) => React.ReactElement<any>;
   renderingOrder: Array<keyof RenderingOrderMapping>;
+  multigraph?: boolean;
```

Comment:
Can we keep multigraph restricted to only DagreReact and not DAGReact?

For example just adding it to DagreReactProps in DagreGraph.tsx.  I don't think it's a concept for other layout engines to worry about.